ra/ra.go

```diff
@@ -787,6 +788,30 @@ func (ra *RegistrationAuthorityImpl) recheckCAA(ctx context.Context, names []str
 	return nil
 }
 
+// failOrder marks an order as invalid by setting the problem details field of
```

Comment:
Perhaps `failed` here instead of `invalid` as it is slightly confusing as to whether this is referring to the order `status` or some nebulous concept of the status of the order.

Reply:
Sounds good to me. Fixed in c126845